lib/geoservices/queryable.rb

```diff
@@ -14,7 +14,16 @@ def query(layer_idx=0, options={})
     # returns nil if the layer does not have the Capability: Query
     def count(layer_idx=0, options={})
       return nil unless self.metadata["capabilities"] =~ /Query/
-      query(layer_idx, options.merge(:returnCountOnly => true))
+      # Current Version is not returned at lower than 10.1
+      unless self.metadata['currentVersion'].nil?
```

Comment:
@dmfenton I specifically meant this check. You are only checking that `if currentVersion.nil?` then use `returnCountOnly`. I would have expected this to be something like `self.metadata['currentVersion'].to_f < 10.1`